modules/python3/src/pythonscript.cpp

```diff
@@ -160,14 +167,14 @@ std::string PythonScript::getAndFormatError() {
                 file = "<script>";
             }
 
-            errstr << file << ":" << line << " in " << name << "\n";
+            err << file << ":" << line << " in " << name << "\n";
             frame = PyFrame_GetBack(frame);
         }
     } else {
-        errstr << "No stacktrace available";
+        err << "No stacktrace available";
     }
 
-    return std::move(errstr).str();
+    return std::move(err).str();
```

Comment:
why std::move on the stringstream?

Reply:
then str() will use the rvalue overload moving the string out with out having to create a copy